packages/gluestick-cli/package.json

```diff
@@ -40,6 +40,7 @@
     "fs-extra": "^2.0.0",
     "gluestick-generators": "^1.0.0-rc.8",
     "glob": "7.1.1",
-    "mkdirp": "0.5.1"
+    "mkdirp": "0.5.1",
+    "shelljs": "0.7.6"
```

Comment:
Don't know how i can get only `which` from `shelljs` but there is another library which do the same without rest from `shelljs` 
https://github.com/mathisonian/command-exists